csp_billing_adapter/adapter.py

```diff
@@ -75,8 +77,8 @@ def get_plugin_manager() -> pluggy.PluginManager:
 def setup_logging() -> logging.Logger:
     """Setup basic logging"""
     logging.basicConfig(
-        format='%(asctime)s.%(msecs)03d|%(levelname)s|%(name)s|%(message)s',
-        datefmt='%Y-%m-%dT%H:%M:%S'
+        format=LOGGING_FORMAT,
+        datefmt=LOGGING_DATEFMT
```

Comment:
Nitpick: underscore for all words. Example: LOGGING_DATE_FMT.

Reply:
No problem
Originally I set it as `LOGGING_DATEFMT`
as `datefmt` is the param in the method
signature, changing it to `LOGGING_DATE_FMT`
for clarity